src/Microsoft.AspNet.OData.Shared/DeltaOfTStructuralType.cs

```diff
@@ -157,6 +220,24 @@ public override bool TryGetPropertyValue(string name, out object value)
                 throw Error.ArgumentNull("name");
             }
 
+            if (_instanceAnnotationsPropertyInfo != null)
+            {
+                if (name == _instanceAnnotationsPropertyInfo.Name)
+                {
+                    if (_instanceAnnotationCache == null)
+                    {
+                        _instanceAnnotationCache =
+                            GetInstanceannotationContainer(_instanceAnnotationsPropertyInfo, _instance, null,create: false);
+                    }
+
+                    if(_instanceAnnotationCache != null)
```

Comment:
>_instanceAnnotationCache != null [](start = 23, length = 32)

if they are asking for the instanceAnnotationCache property, and it's null, shouldn't we return null here?